benchcab/internal.py

```diff
@@ -5,15 +5,16 @@
 
 import os
 from pathlib import Path
-from typing import Any
+
+from benchcab.config import PBSConfig
```

Comment:
I'm not a big fan of the circular dependencies here: internal imports from config and config imports internal.

Should we define PBSConfig in its own file? Or in internal? Or do you prefer to leave it as is?

Reply:
Oops, I didn't see the circular dependency there - will go with Option 1 from before and make `types.py` for now